annotation/core/views.py

```diff
@@ -202,18 +202,39 @@ def save(request):
     text = int(request.POST['text'])
     name = request.POST['name']
     boundary = int(request.POST['boundary'])
-    revision = request.POST['revision']
     points = request.POST['points']
     attention_check = request.POST['attention_check']
 
+    grammar = request.POST['grammar'] == 'true'
```

Comment:
How about defining a constant array 
`feedback_options = ["repetition", "irrelevant", "contradicts_sentence", "contradicts_knowledge", "common_sense", "coreference", "generic"]` and just using a for loop here?

Reply:
Even better, you should just be able to read these from the database with something like
```
feedback_options  = [v[0] for v in FeedbackOption.objects.filter(is_default=True).values_list("shortname")]
```

Reply:
changed!